idaes/core/base/flowsheet_model.py

```diff
@@ -74,10 +74,7 @@ def __init__(self):
             self.visualize = self._visualize_null
             self.installed = False
         else:
-            # FIXME the explicit submodule import is needed because the idaes_ui doesn't import its fv submodule
-            # otherwise, you get "AttributeError: module 'idaes_ui' has no 'fv' attribute"
-            import idaes_ui.fv
-
+            import idaes_ui
```

Comment:
Any reason why this line (and the accompanying comment) are being changed? IIRC that was written by me. I don't remember the details other than what is written there, and I do remember it would cause an error otherwise.

Reply:
nope. I didn't mean to include that file at all